.github/workflows/ci.yml

```diff
@@ -9,21 +9,16 @@ jobs:
     runs-on: ubuntu-latest
     name: cargo fmt
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@v1
         with:
           toolchain: 1.68
```

Comment:
We're not releasing in the near future, so I don't think it makes sense for us to maintain an MSRV

Reply:
I was just keeping what was there, but I can remove these as part of this PR.